assets/js/modules/adsense/datastore/service.js

```diff
@@ -126,8 +125,21 @@ export const selectors = {
 		return select( STORE_NAME ).getServiceURL( { path, query } );
 	} ),
 
+	/**
+	 * Returns the service URL to an AdSense account's site overview page.
```

Comment:
This should describe a bit more what this URL is for. Right now it is an exact copy of the other doc-block description. Should be more like `Returns the service URL to an AdSense report.